src/screens/Game/Gameplay.jsx

```diff
@@ -194,7 +194,7 @@ function Gameplay({ route, navigation }) {
 
       setTimeout(() => {
         getNewProblem();
-      }, 500);
+      }, 5000);
```

Comment:
What's the reasoning for having a timeout of 5000ms? A 5s delay between questions seems way too long